packages/Algernon.package/ALGNullItem.class/instance/name.st

```diff
@@ -1,3 +1,3 @@
 accessing
 name
-^ 'Null Item'
\ No newline at end of file
+	^ 'Null Item'
```

Comment:
Are you sure there should be a tab here?

Maybe it's just github displaying this weirdly.

Reply:
All other return statements are also indented with a tab, therefore I don't see why there shouldn't be one.

Reply:
Okay, thank you, then it's just github.